pkg/functions/client_test.go

```diff
@@ -1532,83 +1532,3 @@ func TestClient_CreateMigration(t *testing.T) {
 		t.Fatal("freshly created function should have the latest migration")
 	}
 }
-
-// TestClient_BuiltDetects ensures that the client's Built command detects
-// filesystem changes as indicating the function is no longer Built (aka stale)
-// This includes modifying timestamps, removing or adding files.
-func TestClient_BuiltDetects(t *testing.T) {
```

Comment:
This test was just moved in its entirety to the functions test, since the `Built` logic is now a method on Function rather than a package static function.